pkg/cmd/accept/exec.go

```diff
@@ -24,11 +24,13 @@ import (
 const (
 	groupNameBootstrap               = "system:bootstrappers:managedcluster"
 	userNameSignatureBootstrapPrefix = "system:bootstrap:"
-	userNameSignatureSA              = "system:serviceaccount:open-cluster-management:cluster-bootstrap"
+	userNameSignatureSAPrefix         = "system:serviceaccount:"
```

Comment:
I do not think we should change the sa name here. It is to ensure that only specifit sa can be approved